kiliautoml/models/_base_model.py

```diff
@@ -105,6 +108,24 @@ def _check_compatible(request: Optional[T], list_ok: List[T], param_name: str) -
                 f"You requested {param_name} {request} but only {list_ok} is available."
             )
 
+    def _check_compatible_model(self, model_name: Optional[ModelNameT]) -> None:
```

Comment:
this should be an abstract method of the BaseModel, with this implementation in the HuggingFace models. Ping me if you need help on this.